### PR TITLE
Test base selectors

### PR DESCRIPTION
**Pull request by ufocoder**
#43

### CODE REVIEW

File: src/selectors/index.js [hunk 1]
```diff
@@ -15,15 +15,16 @@ import {
   combNetworkMSBNDescription,
 } from './combiners';
 
-export const getNetwork = state => state.network;
+export const getNetwork = prop('network');
 export const getNodes = state => state.network.nodes || state.nodes || [];
```
Comment (nolleto) [inline src/selectors/index.js]: You can you [either](https://ramdajs.com/docs/#either) here:
```js
export const getNodes = either(path(['network', 'nodes']), propOr([], 'nodes'));
```

File: src/selectors/index.js [hunk 2]
```diff
@@ -15,15 +15,16 @@ import {
...
 export const getPositions = state => state.network.positions || state.positions || [];
```
Comment (nolleto) [inline src/selectors/index.js]:
```js
export const getPositions = either(path(['network', 'positions']), propOr([], 'positions'));
```

File: src/selectors/index.test.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    });
+  });
+
+  describe('getSubnetworks selector', () => {
+    const subnetworks = [1, 2, 3];
+    const network = { subnetworks };
+    const store = { network };
+    it('gets subnetworks', () => {
+      expect(getSubnetworks(store)).toEqual(subnetworks);
+    });
```
Comment (nolleto) [inline src/selectors/index.test.js]: This selector has two behaviors:
- When network has subnetworks
getSubnetworks = state => **state.network.subnetworks** || []
- When network has no subnetworks
getSubnetworks = state => state.network.subnetworks || **[]**

So, I would update this test to:
```js
describe('getSubnetworks selector', () => {
  desribe('When network has subnetworks', () => {
    const subnetworks = [1, 2, 3];
    const network = { subnetworks };
    const store = { network };

    it('gets subnetworks', () => {
      expect(getSubnetworks(store)).toEqual(subnetworks);
    });
  });

  desribe('When network has no subnetworks', () => {
    const network = { };
    const store = { network };
    
    it('gets subnetworks', () => {
      expect(getSubnetworks(store)).toEqual(subnetworks);
    });
  });
});
```

File: src/selectors/index.test.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      expect(getNetworkKind(store)).toEqual(kind);
+    });
+
+    it('gets default network kind when it not exists', () => {
+      expect(getNetworkKind({})).toEqual(NETWORK_KINDS.BN);
+    });
+  });
+
+  describe('getPanelVisibility selector', () => {
+    const propertiesPanelVisible = 42;
```
Comment (nolleto) [inline src/selectors/index.test.js]: Can you change this to a `Boolean`?
```suggestion
    const propertiesPanelVisible = true;
```

File: src/selectors/index.js (hunk 1 above)
Comment (ufocoder) [reply src/selectors/index.js]: Great, thx

**Comment by ufocoder** [comment]
@nolleto I've updated PR, check it please!

**Comment by nolleto** [comment]
@ufocoder can you perform a `git rebase`?

File: src/selectors/index.test.js [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      expect(getNetwork(store)).toEqual(network);
+    });
+  });
+
+  describe('getNodes selector', () => {
+    describe('When state has nodes', () => {
+      const nodes = [1, 2, 3];
+      const store = { nodes };
+
+      it('gets network nodes', () => {
```
Comment (nolleto) [inline src/selectors/index.test.js]: `gets state nodes` 💅

File: src/selectors/index.test.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+      });
+    });
+  });
+
+  describe('getPositions selector', () => {
+    describe('When state has positions', () => {
+      const positions = [1, 2, 3];
+      const store = { positions };
+
+      it('gets network positions', () => {
```
Comment (nolleto) [inline src/selectors/index.test.js]: `gets state positions` 💅

**Pull request merged**